packages/api-explorer/__tests__/Params.test.jsx

```diff
@@ -240,16 +240,16 @@ describe('oneOf/anyOf', () => {
 
 describe('schema handling', () => {
   describe('minLength / maxLength', () => {
-    it('should put a `min` and `max` attribute on an input', () => {
+    it('should put a `minLength` and `maxLength` attribute on an input', () => {
```

Comment:
`minLength` and `maxLength` were being incorrect mapped to `min` and `max` but that's now fixed in https://github.com/readmeio/oas/pull/358.